geoportailv3/static/js/usercontroller.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * @fileoverview This file defines the controller class for the application's
+ * user controller.
+ *
+ */
+goog.provide('app.UserController');
+
+goog.require('app');
+
+
+
+/**
+ * @param {angular.$http} $http Htpp.
+ * @constructor
+ * @export
+ * @ngInject
+ */
+app.UserController = function($http) {
+
+  /**
+   * @type {object}
+   * @private
+   */
+  this['credentials'] = {
+    'login' : null,
+    'password' : null
+  };
+
+  /**
+   * @type {angular.$http}
+   * @private
+   */
+  this['http_'] = $http;
```

Comment:
You don't need/want to use the bracket notation for `http_` because that property is not used in the view. Using the bracket notation and quotes prevent renaming. We want to use it only when necessary.